pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -1131,7 +1148,10 @@ def test_td64arr_div_int(self, box_with_array):
         result = idx / 1
         tm.assert_equal(result, idx)
 
-    def test_tdi_div_tdlike_scalar(self, two_hours, box_with_array):
+        with pytest.raises(TypeError):
+            1 / idx
```

Comment:
Unless the error message is truly meaningless, `pytest.raises(<error-cls>, match=<msg>)` should definitely be preferred over just `pytest.raises(<error-cls>)`.  This applies to all of your `pytest.raises` checks.